AppleStore/AppleStore/PageVC/InfoMessageHelper.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  InfoMessageHelper.swift
+//  AppleStore
+//
+//  Created by Maksim on 15.07.2021.
+//
+
+struct InfoMessageHelper {
+    var nameImage = String()
```

Comment:
```suggestion
    var imageName = String()
```
заканчивается на Image это же стринги